### PR TITLE
chore: (monorepo) refactor

### PR DESCRIPTION
**Pull request by vegarringdal**
@nchanged 

I changed and added a lot.
Feel free to remove stuff if you think its too much :-)

Though it could be nice with build script/publish to get started. Not perfect but a start.

### CODE REVIEW

**Comment by vegarringdal** [comment]
btw, you can test it right away:

https://codesandbox.io/s/github/fuse-box/sandbox/tree/vr-mono-repo/monorepo

Thats cool

**Comment by vegarringdal** [comment]
wait, I need to update more.. do not merge

**Comment by vegarringdal** [comment]
Done :-)

File: monorepo/scripts/syncPackages.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const { logInfo, readFile, writeFile, readFiles, path } = require('./utils');
+const prettier = require('prettier');
+
+const mainPackage = './package.json';
+const packages = [
+    /*'./package-lock.json'*/
+];
+
+const main = async function() {
+    let mainVersion = JSON.parse(await readFile(mainPackage)).version;
```
Comment (nchanged) [inline monorepo/scripts/syncPackages.js]: man, what's all this? ;-) this reminds me of webpack ;-)

File: monorepo/scripts/utils.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
+// @ts-check
+const fs = require('fs');
+const path = require('path');
+const spawn = require('child_process').spawn;
+
+const renameFolder = (oldPath, newPath) => {
```
Comment (nchanged) [inline monorepo/scripts/utils.js]: why? i we have sparky. I had 5 lines of config man ;-) you replaced it with 5000 ;-)

File: monorepo/scripts/utils.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            if (err) {
+                reject(err);
+            } else {
+                resolve(files);
+            }
+        });
+    });
+};
+
+const writeFile = (file, data) => {
```
Comment (nchanged) [inline monorepo/scripts/utils.js]: I really don't get the purpose of it. I wanted to show how simple it is use to use monorepo with fuse, primarily with a focus on efficient configs. This doesn't seem efficient to me ;-)

File: monorepo/scripts/syncPackages.js (hunk 1 above)
Comment (vegarringdal) [reply monorepo/scripts/syncPackages.js]: removed, was me getting carried away :-)

File: monorepo/scripts/utils.js (hunk 3 above)
Comment (vegarringdal) [reply monorepo/scripts/utils.js]: removed, was me getting carried away :-)

File: monorepo/scripts/utils.js (hunk 2 above)
Comment (vegarringdal) [reply monorepo/scripts/utils.js]: removed, was me getting carried away :-)

**Comment by vegarringdal** [comment]
Live demo of branch.
https://codesandbox.io/s/github/fuse-box/sandbox/tree/vr-mono-repo/monorepo

**Pull request merged**